src/pages/features/gherkin.mdx

```diff
@@ -3,15 +3,15 @@ import Layout from '../../layouts/Layout.astro'
 <Layout title="Gherkin">
 	# Gherkin validation
 
-    **vitest-cucumber** use its own feature file parser and validate it before 
-    run unit tests.
+    **vitest-cucumber** uses its own feature file parser and validate it before 
+    running unit tests.
 
-    And checl you test all scenario, steps etc.
+    And check ths tests for all scenario, steps, etc...
```

Comment:
Not `...` with etc.

Reply:
That was my preference but it seems to be `etc.` indeed.